tsconfig.json

```diff
@@ -24,6 +24,7 @@
       {
         "name": "typescript-strict-plugin"
       }
-    ]
+    ],
+    "useDefineForClassFields": false
```

Comment:
Was there a specific reason we needed to set this to false? I think true is the default for es2022 and it makes i compatible with the standard going forward. Was just curious if there were any errors when this is true?

Reply:
https://www.typescriptlang.org/tsconfig#useDefineForClassFields